bin/node/runtime/src/lib.rs

```diff
@@ -558,9 +563,23 @@ sp_npos_elections::generate_solution_type!(
 pub const MAX_NOMINATIONS: u32 =
 	<NposCompactSolution16 as sp_npos_elections::CompactSolution>::LIMIT as u32;
 
+/// The numbers configured here should always be more than the the maximum limits of staking pallet
```

Comment:
Does it correct for us?

Reply:
Yes, our values are orders of magnitude less.
[This discussion](https://github.com/paritytech/substrate/pull/9286#discussion_r666421141) might make it a bit cleaner